src/runner.ts

```diff
@@ -66,15 +75,51 @@ export const runner = async () => {
       process.exit(1);
```

Comment:
also here we would need something

Reply:
7997a9e705381d15529ac7dc9a910e743e727e60 added the telemetry send in cases where we exit process earlier